### ISSUE TITLE
Nim's hash is wrong

### ISSUE THREAD

**Issue by TiZ-EX1** [Linuxbrew/homebrew-core#6557]
```
WC130-TiZ:~$ brew upgrade nim
==> Upgrading 1 outdated package, with result:
nim 0.18.0
==> Upgrading nim 
==> Downloading https://nim-lang.org/download/nim-0.18.0.tar.xz
Already downloaded: /xusr/linuxbrew/.cache/Homebrew/nim-0.18.0.tar.xz
Error: SHA256 mismatch
Expected: 2ebe164eb4638a7a65aa02da7bdc4d7559026105655fdf3f620b9525406a46c1
Actual: 45c74adb35f08dfa9add1112ae17330e5d902ebb4a36e7046caee8b79e6f3bd0
Archive: /xusr/linuxbrew/.cache/Homebrew/nim-0.18.0.tar.xz
To retry an incomplete download, remove the file above.
```

It seems like a huge pain to try and do a fork/commit/push/PR for such a minor change especially since my linuxbrew is owned by another user for safety. I think just telling you will be less trouble for both of us.

**Comment by iMichka** [comment]
Hi; Thanks for reporting.

I saw this problem here: #6398
I always report it to upstream, https://github.com/Homebrew/homebrew-core, because it is there where it needs to be fixed.

I gave up on running after each project which breaks their releases. This is a security feature in case the projects server got hacked and the file was replaced.

So unless somebody really needs a fix, I have no time to contact all these projects (and this happens about twice a week).

The good news is that somebody on mac saw the problem and fixed the sha, and this just got merged: #6560. I am also building a bottle here: #6561
